src/atoms/icons/index.tsx

```diff
@@ -26,6 +27,7 @@ export default function Icons({ icon }: Props) {
       {icon === 'github' && <GitHub />}
       {icon === 'copy' && <Copy />}
       {icon === 'check' && <Check />}
+      {icon === 'send' && <Send />}
```

Comment:
Can we make this more dynamic? What happens if tomorrow we have to add 100 new icons? This interface will get bigger.